tools/mapmerge2/travis_mapcheck.bat

```diff
@@ -0,0 +1,3 @@
+@echo on
+python travis_mapcheck.py
```

Comment:
also, this script is useless, because you hardcoded the path in the mapcheck file.

Reply:
I accidentally clicked resolved. Well no, the path never changes on travis. And it check for maps folder. I can make it use environment variables. How is it useless though ?

Reply:
The script is intended to run **ONLY** on Travis, it is not designed to use Windows or for user to use it. The windows way of using it is to just run mapmerge process on a map.

This PR is just making mapmerge being run on Travis to check for map problems, it doesn't require to be adapted to windows